modin/pandas/dataframe.py

```diff
@@ -758,39 +758,22 @@ def cov(
                 ddof=ddof,
                 numeric_only=numeric_only,
             )
-        numeric_df = self.drop(
+        cov_df = self.drop(
             columns=[
                 i for i in self.dtypes.index if not is_numeric_dtype(self.dtypes[i])
             ]
         )
 
-        is_notna = True
+        if min_periods is not None and min_periods > len(cov_df):
```

Comment:
Why are we able to take this much simpler codepath? Seems that the previous code was doing the computation by hand (probably because we `pandas` behavior with NaN's was non ideal), but now we can do it automatically - perhaps because pandas changed how something is implemented?